src/main/scala/it/unibo/authsim/library/user/builder/util/Util.scala

```diff
@@ -0,0 +1,16 @@
+package it.unibo.authsim.library.user.builder.util
+
+import scala.util.Random
+
+/**
+ * Object for usefull functions
+ */
+object Util {
+  /**
+   * Method that generate a randomic string composed by letter and numbers
+   * @param length    length of the string to be created, default value is 5
+   * @return          new randomic string of the given length 
+   */
+  def generateRandomString(length:Int=5): String=
+    Random.alphanumeric.filter(_.isLetterOrDigit).take(length).mkString
```

Comment:
`Random.alphanumeric` and `Random.alphanumeric.filter(_.isLetterOrDigit)` should return the same stream, shouldn't they? Is the filtering necessary?

Also, check if `filter(_.isLetterOrDigit)` can be written `filter(isLetterOrDigit)`, as it is a bit simpler and readable.